### PR TITLE
Ch36 career counseling claim

### PR DESCRIPTION
**Pull request by digitaldrk**
## Description of change
We are migrating the [Chapter36 form from](https://www.vba.va.gov/pubs/forms/VBA-28-8832-ARE.pdf) eBenefits to va.gov 😎 . This is a fairly simple form that upon successful submission, will be sent to CentralMail. _Please let me know if you would like to break this PR in to smaller ones, I would be happy to do that_.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Things to know about this PR
* This code leverages the `CentralMailClaim` model to handle central mail submissions. Upon discovery of that model and associated services, we had to make some schema changes.
* I am currently getting a 401 unauthorized error upon sending to Centra Mail. I think it is because I don't have a token for central mail locally, I have asked about this in #vfs-platofrm-support.
* Are there additions to a `settings.yml` file? Do they vary by environment?
  * no new settings
* Is there a feature flag? What is it?
  * This code will not be accessible to the public
* Is there some Sentry logging that was added? What alerts are relevant?
  * There is a raven tag in the controller that checks for claim save.
* Are there any Prometheus metrics being collected? What Grafana dashboard were they added do?
  * No prometheus was added
* Are there Swagger docs that were updated?
  * Swagger was updated
* Is there any PII concerns or questions?
  * No Concerns

### CODE REVIEW

**Comment by LindseySaari** [comment]
Did you get an answer to this issue yet? "I am currently getting a 401 unauthorized error upon sending to Centra Mail. I think it is because I don't have a token for central mail locally, I have asked about this in #vfs-platofrm-support."

File: app/controllers/v0/education_career_counseling_claims_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module V0
+  class EducationCareerCounselingClaimsController < ClaimsBaseController
+    def create
```
Comment (LindseySaari) [inline app/controllers/v0/education_career_counseling_claims_controller.rb:5]: Do you have a spec for this new controller method?

Comment (digitaldrk) [reply app/controllers/v0/education_career_counseling_claims_controller.rb:5]: @LindseySaari I have added some specs. Thank you for bringing this up 🤘🏻

**Comment by digitaldrk** [comment]
> Did you get an answer to this issue yet? "I am currently getting a 401 unauthorized error upon sending to Centra Mail. I think it is because I don't have a token for central mail locally, I have asked about this in #vfs-platofrm-support."

@LindseySaari Negative, no answer back in platform support. I will ask again. LMK if you have other avenues.

**Review by LindseySaari — APPROVED** [review]
Thanks for the changes! Looks good 👍

**Pull request merged**